packages/jetpack-blocks/src/blocks/simple-payments/utils.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { trimEnd } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrencyDefaults } from '../../../../../client/lib/format-currency/currencies';
```

Comment:
Not OK! Needs #31229 
```suggestion
import { getCurrencyDefaults } from '@automattic/format-currency/currencies';
```

Reply:
Fixed by the rebase.

Reply:
3d6cb23109